.github/workflows/submit.yml

```diff
@@ -0,0 +1,46 @@
+name: Kodi Addon-Submitter
+
+on:
+  create:
+    tags:
+      - v*
```

Comment:
If all is merged, you will trigger this by pushing a tag that starts with `v` for e.g. `v1.0.0`